README.md

```diff
@@ -24,6 +24,19 @@ See [rust-lang/rust#35437][0].
 [2]: https://github.com/rust-lang/compiler-rt/tree/8598065bd965d9713bfafb6c1e766d63a7b17b89/lib/builtins
 [3]: https://travis-ci.org/japaric/rustc-builtins
 
+### Porting Reminders
+
+1. [Rust][4] and [C][5] have slightly different operator precedence.
+2. C assumes wrapping operations everywhere. Rust panics on overflow when in debug mode. Consider using the [Wrapping][6] type or the explicit [wrapping_*][7] functions where applicable.
+3. Note [C implicit casts][8], especially integer promotion. Rust is much more explicit about casting, be sure that any cast which affects the output is ported to the Rust implementation.
+4. Rust has many functions for integer or floating point manipulation in the standard library. Consider using one of these functions rather than porting a new one. 
```

Comment:
Could you add links to the primitive doc pages like https://doc.rust-lang.org/std/primitive.i8.html ?